WildAlertApi/WildAlert.UnitTests/CreateAlertCommandHandlerTests.cs

```diff
@@ -0,0 +1,53 @@
+using FluentAssertions;
+using MapsterMapper;
+using Microsoft.EntityFrameworkCore;
+using WildAlert.Application.Requests.Alerts.Commands.CreateAlert;
+using WildAlert.Persistence;
+using WildAlert.Persistence.Entities.Alert;
+using WildAlert.Shared.DateTimeProvider;
+using WildAlert.Tests.Shared.DateTimeProvider;
+
+namespace WildAlert.UnitTests;
+
+public class CreateAlertCommandHandlerTests
+{
+    private ApplicationDbContext _context = null!;
+    private IMapper _mapper = null!;
+    private IDateTimeProvider _dateTimeProvider = null!;
+    
+    [SetUp]
+    public void Setup()
+    {
+        var options = new DbContextOptionsBuilder<ApplicationDbContext>()
+            .UseInMemoryDatabase(Guid.NewGuid().ToString())
+            .Options;
+        
+        _context = new ApplicationDbContext(options);
```

Comment:
We should make something like `ApplicationDbContextFactory`. We can dissuss that later